BrainPortal/app/controllers/sessions_controller.rb

```diff
@@ -147,7 +147,7 @@ def create_from_user(user) #:nodoc:
     end
 
     self.current_user = user
-    session[:user_id] = user.id if request.format.to_sym == :html
+    session[:user_id] = user.id  if request.format.to_sym == :html
```

Comment:
extra space needed?

Reply:
Yep, it's a mistake from me.

Reply:
I noticed it too after I pushed the code.